src/main/resources/DDL.sql

```diff
@@ -0,0 +1,30 @@
+CREATE DATABASE my_ticket_service
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LOCALE_PROVIDER = 'libc'
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
+
+CREATE TABLE public."User"
+(
+    id character varying NOT NULL,
```

Comment:
use 
```sql
id varchar (50) not null
```
instead of 
```sql
 id character varying NOT NULL
```

Reply:
okay, my DDL was generated by Postgres automatically when I chose the required columns. I will create it from scratch using plain SQL.